### ISSUE TITLE
AdGuard Home Binding Port on IPv6 Only

### ISSUE THREAD

**Issue by Haravikk** [AdguardTeam/AdGuardHome#7025]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Darwin (aka macOS), AMD64 (aka x86_64)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

v0.107.50

### Action

```sh
sudo lsof -PiTCP -sTCP:LISTEN | grep AdGuard
```

### Expected result

AdGuardHome should be bound port 3000 on both IPv4 and IPv6

### Actual result

AdGuardHome is only bound on port 3000 on IPv6, making it inaccessible by other machines on an IPv4 network:
```
AdGuardHo  160   root   99u  IPv6 0x9b74b5665d50fa17      0t0  TCP *:3000 (LISTEN)
AdGuardHo  160   root  104u  IPv6 0x9b74b5665d510657      0t0  TCP *:53 (LISTEN)
```

### Additional information and/or screenshots

Not sure when this issue started occurring as I'm not sure what version I was on prior to updating, but it remains an issue; I have to go to each machine using AdGuard Home and connect using `localhost:3000` locally, rather than connecting remotely as I normally would.

Here's what the top of my `AdGuardHome.yaml` file looks like on one affected machine:

```
http:
  pprof:
    port: 6060
    enabled: false
  address: 0.0.0.0:3000
  session_ttl: 720h
```

Seems like it should be correct, and I'm not seeing an options anywhere to configure for IPv6 only – if anything the address given would indicate IPv4 only surely?

**Comment by nathanialhenniges** [comment]
Am having this same issue.

**Comment by zhangguanzhang** [comment]
any update?
```
http:
  pprof:
    port: 6060
    enabled: false
  address: "[::]:3000"
```

[error] parsing configuration file: dns.bind_hosts at index 0 is not a valid ip address

**Comment by pieterburm** [comment]
anyone find what is wrong and have a solution?

**Comment by DarkG3n3sis** [comment]
same problem here.... i found that problem when qbittorrent stop working one day. i got this after an update.... . i previusly have installed this https://aur.archlinux.org/packages/adguardhome-bin by the aur, recently some pp added an official one on arch and i try to nuke my install of adg and installed the official one and on setup same problem, i port the confing.yaml after test and same problem obv. Is realy annoyng

**Comment by Ducki** [comment]
Can confirm for the docker image as well.

**Comment by AchtungMoose** [comment]
downgrading to v0.107.57 resolves this. 
problem occurs from v0.107.58 onwards

**Comment by gtxaspec** [comment]
Hi, any update on this? @ainar-g  any insight if this would be supported? force web port on ipv6 only? 

it looks to support an ipv4 address only for `address` under the `http` category at the moment.

**Comment by uqs** [comment]
Indeed, trying to get the web UI bound to an IPv6 address trips up the parser:

May 15 14:00:04 ohpi.spoerlein.net AdGuardHome[9198]: 2026/05/15 14:00:04.678543 [info] starting adguard home version="AdGuard Home, version v0.107.74"
May 15 14:00:04 ohpi.spoerlein.net AdGuardHome[9198]: 2026/05/15 14:00:04.679328 [info] service: AdGuard Home, version v0.107.74
May 15 14:00:04 ohpi.spoerlein.net AdGuardHome[9198]: 2026/05/15 14:00:04.679379 [info] service: control action=run
May 15 14:00:04 ohpi.spoerlein.net AdGuardHome[9198]: 2026/05/15 14:00:04.684360 [error] failed to parse configuration file err="yaml: construct errors: line 12: ParseAddr(\"tcp://[2a02:168:5230::161]\"): each colon-separated field must have at least one digit (at \"tcp://[2a02:168:5230::161]\")"


I also tried w/o the tcp:// prefix. A shame only legacy IP is supported :(

**Comment by uqs** [comment]
I take that back, the syntax is just odd:

works:  address: '[2:xxx::1]:80'
doesn't work: address: [2:xxx:1]:80
